fenix/app/src/main/java/org/mozilla/fenix/shopping/store/ReviewQualityCheckAction.kt

```diff
@@ -214,9 +219,4 @@ sealed interface ReviewQualityCheckAction : Action {
      * Triggered when the No analysis card is displayed to the user.
      */
     object NoAnalysisDisplayed : TelemetryAction
-
-    /**
-     * Triggered when the user reports a product is back in stock.
-     */
-    object ReportProductBackInStock : TelemetryAction
```

Comment:
Let's use the existing action

Reply:
Updated to use action `ReportProductBackInStock`